stripe/src/main/java/com/stripe/android/AbstractEphemeralKey.java

```diff
@@ -244,13 +239,9 @@ String getType() {
         } catch (InvocationTargetException e) {
             throw new IllegalArgumentException("Exception instantiating " + ephemeralKeyClass, e);
         } catch (NoSuchMethodException e) {
-            throw new IllegalArgumentException("Class " + ephemeralKeyClass + " does not have an accessible (JSONObject) constructor", e);
-        } catch (Exception e) {
-            if (e instanceof JSONException) {
-                // ignored
-                return null;
-            }
-            throw e;
+            throw new IllegalArgumentException("Class " +
```

Comment:
you can remove the `@SuppressWarnings` annotation on this method now

Reply:
👍